### PR TITLE
{lib}[foss/2021b] uncertainty-calibration v0.0.9 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3162.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/39b921dc91eacc57e2e15c2261cbf730 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/e38ed69d0d9911cbc82420152360c079 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/22b8e193dcd5796c86dc58a3c7e19ee6 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04.2 LTS (Focal Fossa), x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/405c0ad5d21f84f415ddb4a53bffef8f for a full test report.

File: easybuild/easyconfigs/u/uncertainty-calibration/uncertainty-calibration-0.0.9-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('scikit-learn', '1.0.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('sklearn', '0.0', {
```
Comment (branfosj) [inline easybuild/easyconfigs/u/uncertainty-calibration/uncertainty-calibration-0.0.9-foss-2021b.eb:21]: I dislike this and I am wondering if #15613 would be a better solution.

Comment (boegel) [reply easybuild/easyconfigs/u/uncertainty-calibration/uncertainty-calibration-0.0.9-foss-2021b.eb:21]: Oh yes, I think the approach in #15613 makes sense...

Comment (boegel) [reply easybuild/easyconfigs/u/uncertainty-calibration/uncertainty-calibration-0.0.9-foss-2021b.eb]: fixed in 2b62347

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**